src/gg.regression.unity.bots/Runtime/Scripts/StateRecorder/SdkApiVersion.cs

```diff
@@ -47,9 +47,13 @@ public static class SdkApiVersion
          * <summary>added criteria to wait for completion of bot actions</summary>
          */
         public const int VERSION_11 = 11;
+        /**
+         * <summary>add code coverage recording support</summary>
+         */
+        public const int VERSION_12 = 12;
```

Comment:
@svolokh  .. You'll now need to make this VERSION_13 and update the files in the this PR using it due to other prs being merged.